### PR TITLE
fix(reports): fixes indent on operating account

### LINKED ISSUE #4484: Compte d'Exploitation should indent child accounts of parents
In the following image, the highlighted account (641399) is the parent of all the child accounts below it (6413XX).  
image

You would never know this from the report.  In the account management module, we indent the accounts to show the parent/child relationship.   We should do the same thing for this report.  WE should also put the parents in bold.

### PR DESCRIPTION
**Pull request by jniles**
Fixes the indentation on the operating account report.  It also makes all titles bold.

Closes #4484.

Here is what it looks like: 
image

### CODE REVIEW

**Comment by jniles** [comment]
@jeremielodi, can I get a review?

File: server/controllers/finance/reports/operating/index.js [hunk 1]
```diff
@@ -23,31 +18,22 @@ const INCOME_ACCOUNT_TYPE = 4;
...
+ * Renders the Compte d'Exploitation
+ *
  * @param {*} options the report options
  * @param {*} session the session
  */
-function reporting(opts, session) {
-  const params = opts;
-  let docReport;
-  const options = _.extend(opts, {
+async function reporting(params, session) {
```
Comment (jeremielodi) [inline server/controllers/finance/reports/operating/index.js:29]: Do you think it's a good idea to use this async without a ```try catch```?

**Review by jeremielodi — APPROVED** [review]

File: server/controllers/finance/reports/operating/index.js (hunk 1 above)
Comment (jniles) [reply server/controllers/finance/reports/operating/index.js:29]: The try/catch occurs in the function that calls this one - on this line: https://github.com/IMA-WorldHealth/bhima/pull/4487/files#diff-4c6917cb3623fce5a08ff0fa7b067112R153

Comment (jeremielodi) [reply server/controllers/finance/reports/operating/index.js:29]: okay

**Pull request merged**